pkg/chart/chart.go

```diff
@@ -320,7 +320,7 @@ func (c *Chart) InitNetClient(details *Details) (HTTPClient, error) {
 
 	defaultHeaders := http.Header{"User-Agent": []string{c.userAgent}}
 	if auth.Header != nil {
-		secret, err := c.kubeClient.Core().Secrets(namespace).Get(auth.Header.SecretKeyRef.Name, metav1.GetOptions{})
+		secret, err := c.kubeClient.CoreV1().Secrets(namespace).Get(auth.Header.SecretKeyRef.Name, metav1.GetOptions{})
```

Comment:
why this change?

Reply:
I don't know. I think you suggested it. (Do note that it's part of 1e1a3ba7c4c3c7b0a177adcffec07169847c1a84.)

Reply:
I guess that was me trying things, you don't need this change